chatcommands.lua

```diff
@@ -362,26 +362,33 @@ minetest.register_chatcommand("area_info", {
 			size_limit_high or size_limit
 
 		-- Privilege information
-		local self_prot_line = ("Self protection is %sabled"):format(
-				self_prot and "en" or "dis")
-		if self_prot and prot_priv then
-			self_prot_line = self_prot_line..
-				(" %s have the neccessary privilege (%q).")
-				:format(
-					has_prot_priv and "and you" or
-						"but you don't",
-					prot_priv)
-		else
-			self_prot_line = self_prot_line.."."
-		end
+		local self_prot_line = (self_prot and prot_priv) and
+			(has_prot_priv and
+				(self_prot and
+				S("Self protection is enabled and you have the "..
```

Comment:
Here it might be simpler to split this into two rather than four strings:
- `S("Self protection is enabled.")`
- `S("You have the necessary privilege (\"@1\").")`